.github/workflows/try-runtime.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  schedule:
+    - cron: '0 6 * * 0' # At 00:00 on Sunday
+  pull_request:
+    types: [labeled]
+    paths:
+      - '**.rs'
+      - .github/workflows/try-runtime.yml
+jobs:
+  try-runtime:
+    strategy:
+      matrix:
+        chain: [altair, centrifuge]
+    if: github.event_name == 'schedule' || contains(github.event.pull_request.labels.*.name, 'D8-migration')
+    runs-on: ubuntu-latest-4-cores
+    name: try-runtime "${{ matrix.chain }}"
+    steps:
+      - name: Check out code
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab #3.5.2
+
+      - name: Prep build on Ubuntu
+        uses: ./.github/actions/prep-ubuntu 
+
+      - name: Install try-runtime-cli
+        run: cargo install --git https://github.com/paritytech/try-runtime-cli --tag v0.5.4 --locked
```

Comment:
Why is this needed?

Reply:
prep-ubuntu peforms `scripts/install_toolchain.sh`. Is that enough ?
I'll try removing it